setup.py

```diff
@@ -18,7 +18,7 @@
       ],
       py_modules=['tap_snowflake'],
       install_requires=[
-          'pipelinewise-singer-python==1.*',
+          'pipelinewise-singer-python==2.0.1',
```

Comment:
Do we have any dependency conflicts here similar to the case we had perviously? Just to double check :)

Reply:
@jayma91 - i don't think so - I've started the practice of destroying the virtualenv of the python projects prior to lib version changes like this to avoid the issue we had earlier.